.github/workflows/CD.yml

```diff
@@ -165,11 +165,12 @@ jobs:
           aws-secret-access-key: ${{ secrets.INTEG_TEST_AWS_KEY_SECRET }}
           aws-region: us-west-2
 
-      - name: Set up JDK 1.11
+      - name: Set up JDK 11
         if: steps.s3-release-validation-1.outputs.cache-hit != 'true'
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v2
         with:
-          java-version: 1.11
+          distribution: 'zulu'
```

Comment:
Do we need to specify distribution

Reply:
Yes. Since the setup-java@v1 has a default options of ```Zulu distribution```. However, based on [setup-java@v2 documents](https://github.com/aws-observability/aws-otel-collector/pull/820), we need to specify the distribution.